prerendering.bs

```diff
@@ -545,6 +550,15 @@ Patch the [=navigate=] algorithm to prevent certain navigations in a [=prerender
   1. Otherwise, if <var ignore>browsingContext</var> is a [=prerendering browsing context=], then return.
 </div>
 
+<div algorithm=The worker's lifetime patch">
+  In <a spec=HTML>The worker's lifetime</a>, modify the definition of [=suspendable worker=], to count workers with a [=prerendering browsing context=] owner as [=suspendable worker|suspendable=]:
+
+  A worker is said to be a [=suspendable worker=] if all of its [=WorkerGlobalScope/owner set|owners=] are {{Document/prerendering}} {{Document|Documents}}, or if it is not an [=active needed worker=] but it is a [=permissible worker=].
```

Comment:
"`prerendering` `Documents`" is not a sufficiently-precise way to say this. I think it'd be better to say "if all of its owners are `Document`s whose browsing contexts are prerendering browsing contexts".

At that point we might want two bullet points instead of a big sentence.

Reply:
Done